x-pack/plugins/ml/public/application/routing/routes/aiops/single_endpoint_streaming_demo.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { parse } from 'query-string';
+
+import { i18n } from '@kbn/i18n';
+
+import { AIOPS_ENABLED } from '@kbn/aiops-plugin/common';
+
+import { NavigateToPath } from '../../../contexts/kibana';
+
+import { MlRoute, PageLoader, PageProps } from '../../router';
+import { useResolver } from '../../use_resolver';
+import { SingleEndpointStreamingDemoPage as Page } from '../../../aiops/single_endpoint_streaming_demo';
+
+import { checkBasicLicense } from '../../../license';
+import { checkGetJobsCapabilitiesResolver } from '../../../capabilities/check_capabilities';
+import { cacheDataViewsContract } from '../../../util/index_utils';
+import { getBreadcrumbWithUrlForApp } from '../../breadcrumbs';
+
+export const singleEndpointStreamingDemoRouteFactory = (
+  navigateToPath: NavigateToPath,
+  basePath: string
+): MlRoute => ({
+  id: 'single_endpoint_streaming_demo',
+  path: '/aiops/single_endpoint_streaming_demo',
+  title: i18n.translate('xpack.ml.aiops.singleEndpointStreamingDemo.docTitle', {
+    defaultMessage: 'Single endpoint streaming demo',
+  }),
+  render: (props, deps) => <PageWrapper {...props} deps={deps} />,
+  breadcrumbs: [
```

Comment:
If you click the AIOps breadcrumb level, this is displayed. It should probably default to the Explain Log Rate Spikes page.

![image](https://user-images.githubusercontent.com/7405507/168557883-bac3b114-14ea-4732-8650-e2214f29cb17.png)

Reply:
Fixed the breadcrumbs in [b39c970](https://github.com/elastic/kibana/pull/132121/commits/b39c97041c85cae2b182f3e2765b2a17d83aab14).